core_screenshot_config.yaml

```diff
@@ -151,7 +151,11 @@ secondary:
       page.done();
     message: waiting 30 sec to load NJ secondary
 
-  PA:
+  OK:
+    file: pdf
+    message: downloading pdf for OK - daily name is autogenerated by formula   
+
+    PA:
```

Comment:
thanks! the OK config looks good but this line got accidentally indented, could you reverse? (it'll break otherwise)

Reply:
oops, fixed. thanks for catching it